src/util/util_errors.h

```diff
@@ -178,4 +178,10 @@ enum sssd_errors {
  */
 const char *sss_strerror(errno_t error);
 
+/* return ldb error converted to an errno */
+errno_t ldb_error_to_errno(int ldberr);
+
+/* define old name for backward compatibility */
+#define sysdb_error_to_errno(ldberr) ldb_error_to_errno(ldberr)
```

Comment:
I would prefer to remove the old name. So for the short term backward compatibility it does not matter much where it is IMO. Do you agree?

Reply:
This will make the diff really huge. Db module is full of this call.

Reply:
I would prefer to drop the name, but I do not have strong opinion on it and I see it can have some downsides when backporting patches to older branches. To move this PR forward I would do the following:
- Move the #define alias to the sysdb.h where the original function was declared
- create an issue to remove it later (maybe before some downstream rebase would be a good time)
- add FIXME comment before the alias is #defined with link to the issue.

We can then discuss in the new issue if and when it makes sense to drop the name.